src/ate/test_programs/dut_lib/dut_lib.h

```diff
@@ -17,17 +17,16 @@ class DutLib {
   void* transport_;
 
  public:
-  DutLib(){};
+  DutLib(void* transport_);
```

Comment:
nit: s/transport_/transport/ as this is a function parameter.

Reply:
oops, copy pasta error; fixed.